trino-connector/integration-test/src/test/resources/trino-ci-testset/trino-cascading-testsets/lakehouse-iceberg-hive/catalog_iceberg_prepare.sql

```diff
@@ -0,0 +1,113 @@
+call gravitino.system.create_catalog(
```

Comment:
Why   don't we reuse these iceberg test cases

Reply:
Reused.